### PR TITLE
made solution generator cross python

### PR DESCRIPTION
**Pull request by vmeshche**
# Description
I have fixed a problem: solution generator support python 3.6+ only. made tool cross python.

Changes proposed in this pull request:
- python 3.6+ specific calls replaced with universal.

### CODE REVIEW

File: deploy/local/generate_win_solution.py [hunk 1]
```diff
@@ -140,9 +140,14 @@ def generate_proj(config, relative_example_path):
 
 def find_all_examples(examples_dir):
     examples = []
-    with cd(examples_dir):
-        for relative_path in glob.iglob('**/*.cpp', recursive=True):
-            examples.append(relative_path)
+
+    print('Walking dir {}'.format(examples_dir))
+    for root, dirnames, filenames in os.walk(examples_dir):
+        print('-> checking file: {}{}'.format(root,filenames))
```
Comment (rlnx) [inline deploy/local/generate_win_solution.py:146]: Is this print really necessary or it was added for debugging? We already have prints in  `generate_sln` and `write_proj`.

Comment (vmeshche) [reply deploy/local/generate_win_solution.py:147]: I will remove it. Let me complete testing

**Pull request merged**